openssl/src/ssl/tests/mod.rs

```diff
@@ -226,10 +237,10 @@ run_test!(new_sslstream, |method, stream| {
     SslStream::connect(&SslContext::new(method).unwrap(), stream).unwrap();
 });
 
-run_test!(get_ssl_method, |method, _| {
-    let ssl = Ssl::new(&SslContext::new(method).unwrap()).unwrap();
-    assert_eq!(ssl.ssl_method(), method);
-});
+// run_test!(get_ssl_method, |method, _| {
```

Comment:
Oh I left this in as I figure we'd want to sort out what to do with the `ssl_method` method, which this was testing.